core/utils_test.py

```diff
@@ -67,8 +67,10 @@ def test_yaml_dict_conversion(self) -> None:
         test_dicts = [{}, {'a': 'b'}, {'a': 2}, {'a': ['b', 2, {'c': 3.5}]}]
 
         for adict in test_dicts:
-            yaml_str = yaml.dump(adict, allow_unicode=True, default_flow_style=False) # type: ignore[no-untyped-call]
-
+            yaml_str = yaml.dump(adict, allow_unicode=True,
+                                 default_flow_style=False)  
+            # type: ignore[no-untyped-call]
```

Comment:
```py
            yaml_str = yaml.dump(adict, allow_unicode=True, # type: ignore[no-untyped-call]
                                 default_flow_style=False) 
```

Reply:
@DiptoChakrabarty  please don't force push if possible

Reply:
Okay will make sure ahead , I had done to rebase my fork